### ISSUE TITLE
addons message spmc 16.6

### ISSUE THREAD

**Issue by janolapin2016** [koying/SPMC#860]
Hey all, Some addons get me this message : Update to krypton because your version doesnt longer support httpS url's...???  I use last SPMC 16.6 on AMLogic S912 Device, Krypton is not good for me. Is that true SPMC doesnt longer support HTTPS addons URL's links ?

**Comment by antonywu** [comment]
Same here, so I switched over to Kodi on my Android. Maybe I will switch back once the issue is resolved

**Comment by janolapin2016** [comment]
For now I think we dont need to switch anywhere. This is not an issue but just a warning to push us to eat krypton, I dont know if this really change anything...

**Comment by s2law** [comment]
It's as it states. Ssl links don't work in Kodi 16 or spmc, but do in 17.
So you'll get more links with Krypton than Jarvis, which is why I'm also
waiting for the SPMC 17. That said, the only thinks missing in Krypton for
me is the voice controls.

**Comment by janolapin2016** [comment]
Wrong. I can run and see all my addons httpS links for now in SPMC and Kodi. I just had this annoying message that in near future this will not be supported. I'm not sure thats concern SPMC ... this is why I ask people ...

**Comment by BigOkie** [comment]
NOt sure about that.  I'm having issues with the opensubtitles addon now, saying I don't have a password/username combo correct.  They JUST switched to HTTPS pretty recently.  Not sure if the that has anything to do with it, but it didn't say anything about https not being supported.

**Comment by janolapin2016** [comment]
yes you right

**Comment by janolapin2016** [comment]
I tried yesterday and my subtitle addon run as hell (opensubtitles) with SPMC 16.6 . All run good for me with 16.6.

**Comment by bravado07** [comment]
From what I could tell 16.6 is affected as well
